app/views/hyrax/admin/admin_sets/_form_visibility.html.erb

```diff
@@ -44,7 +44,7 @@
                   <h3><%= t('.visibility.title') %></h3>
                   <p><%= t('.visibility.description') %></p>
                   <%# List each option in a <div class='radio'> tag %>
-                  <%= f.collection_radio_buttons :visibility, f.object.visibility_options, :first, :last, item_wrapper_tag: :div, item_wrapper_class: 'radio' do |b| %>
+                  <%= f.collection_radio_buttons :visibility, f.object.visibility_options, :first, :last, item_wrapper_tag: :div, item_wrapper_class: 'radio', checked: f.object[:visibility] do |b| %>
```

Comment:
@laritakr could it be that simple_form is by default doing `f.object['visibility']` (with a String, not a Symbol)? Can you check that the form object is returning the correct value if either a string or symbol is passed to the `[]` method?